WordPress/src/main/java/org/wordpress/android/ui/reader/views/ReaderExpandableTagsView.kt

```diff
@@ -200,34 +196,12 @@ class ReaderExpandableTagsView @JvmOverloads constructor(
         @ColorRes
         fun overflowStrokeColorRes(isCollapsed: Boolean): Int? = null
 
-        object Legacy : ChipStyle {
+        data object Default : ChipStyle {
```

Comment:
Now that we have just one implementation, what do you think about making `ChipStyle` a concrete class and moving the implementation from this subclass to there?